src/views/ops/Template/components/Variable.vue

```diff
@@ -4,6 +4,7 @@
       <el-table :data="variables" class="el-table--fit el-table--border">
         <el-table-column show-overflow-tooltip :label="$tc('Name')" prop="name" />
         <el-table-column show-overflow-tooltip :label="$tc('VariableName')" prop="var_name" />
+        <el-table-column show-overflow-tooltip :label="$tc('DefaultValue')" prop="default_value" />
         <el-table-column :label="$tc('Actions')" align="center" class-name="buttons" fixed="right" width="135">
           <template v-slot="scope">
             <el-button icon="el-icon-minus" size="mini" type="danger" @click="removeVariable(scope.row)" />
```

Comment:
The provided code snippet is related to managing variables and operations within a table using ElTable component from the ElementUI framework. To effectively analyze these, please provide the actual content of the original and revised parts (or any discrepancies, if applicable).